indexed-capitalization/solution.js

```diff
@@ -0,0 +1,11 @@
+function capitalize(s,arr){
+  var arrString = s.split('');
+  for (var i = 0; i < arr.length; i++) {
+    if (arr[i] > arrString.length) {
+      continue;
```

Comment:
Whenever you have `continue` and no other code within an `if` statement, then that is a sign that you can reverse the condition, and move the `else` part into the `if` and remove the `else`.